### ISSUE TITLE
NPM package release 0.1.12, removed TypeScript source => broken sourcemap references => breaks Browserify / Sorcery bundler

### ISSUE THREAD

**Issue by danielweck** [rbuckton/reflect-metadata#87]
`0.1.10` contains the original TypeScript (e.g. `Reflect.ts`):
https://unpkg.com/reflect-metadata@0.1.10/

`0.1.12` does not anymore:
https://unpkg.com/reflect-metadata@0.1.12/

This is problematic because (for example) the `Reflect.ts` reference declared in the `Reflect.js.map` sourcemap is broken:
https://unpkg.com/reflect-metadata@0.1.12/Reflect.js.map

...which trips the Browserify + Exorcist + Sorcery build process  (as the transpilation chain from TypeScript to JavaScript cannot be resolved).

**Comment by christophercr** [comment]
Hi, are there any news about this issue?

**Comment by aminere** [comment]
I stuck to reflect-metadata 0.1.10 to make it work (thanks @danielweck), not sure there is anything in 0.1.12 I really needed anyway.

**Comment by danielweck** [comment]
My "workaround" is a bit sucky, but it suits my project fine for now:
`curl https://raw.githubusercontent.com/rbuckton/reflect-metadata/master/Reflect.ts > Reflect.ts && cp Reflect.ts ./node_modules/reflect-metadata/`
:)
(I have this as an additional step in my build CLI / npm scripts)

**Comment by devtobo** [comment]
Since this seems to be fixed in master can we get the npm package published please? Or is there something preventing it?

**Comment by danielweck** [comment]
I am currently updating a bunch of NPM dependencies, so just a quick heads-up: version `0.1.12` of `reflect-metadata` still publishes a sourcemap that references the non-existing origin TypeScript file:
https://unpkg.com/reflect-metadata@0.1.12/
`Reflect.js`
```javascript
//# sourceMappingURL=Reflect.js.map
```
=>
`Reflect.js.map`
```json
"sources":["Reflect.ts"]
```
=>
`Reflect.ts`
boom, file does not exist :)
(I still use the `curl` hack I mentioned above in my build system, to fetch `Reflect.ts` manually and to "force-publish" it into `node_modules`)

**Comment by danielweck** [comment]
Oh, is this fixed now? (`0.1.13`)
https://github.com/rbuckton/reflect-metadata/commit/5b98fe1347f7659f063c6b9301abea89c834d66e
:)

**Comment by danielweck** [comment]
No more source map in `0.1.13`, so this problem is solved.

 https://unpkg.com/reflect-metadata@0.1.13/
